spec/support/connection_string_tests/valid-warnings.yml

```diff
@@ -11,41 +11,41 @@ tests:
                 port: ~
         auth: ~
         options: ~
-    # -
-    #     description: "Unsupported option values are ignored"
-    #     uri: "mongodb://example.com/?fsync=ifPossible"
-    #     valid: true
-    #     warning: true
-    #     hosts:
-    #         -
-    #             type: "hostname"
-    #             host: "example.com"
-    #             port: ~
-    #     auth: ~
-    #     options: ~
-    # -
-    #     description: "Repeated option keys"
-    #     uri: "mongodb://example.com/?replicaSet=test&replicaSet=test"
```

Comment:
I think we still want to test for this error. Only readPreferenceTags can be repeated.